docs/environments.rst

```diff
@@ -43,6 +51,58 @@ files in your config. For example::
       variables:
         InstanceType: ${web_instance_type}
 
+YAML environments
+-----------------
+
+YAML environments allow for more complex environment configuration rather
+than simple text substitution, and support YAML features like anchors and
+references. To build on the example above, let's define a stack that's
+a little more complex::
+
+  stacks:
+    - name: webservers
+      class_path: stacker_blueprints.asg.AutoscalingGroup
+      variables:
+        InstanceType: ${web_instance_type}
+        IngressCIDRsByPort: ${ingress_cidrs_by_port}
+
+We've defined a stack which expects a list of ingress CIDR's allowed access to
+each port. Our environment files would look like this::
+
+  # in the file: stage.env
```

Comment:
shouldn't this doc state stage.yml and not stage.env?

Reply:
Yup, will update shortly!